.github/scripts/check-yaml.py

```diff
@@ -20,6 +20,9 @@
 from referencing.exceptions import NoSuchResource
 from referencing.jsonschema import DRAFT202012
 from referencing.typing import URI
+from urllib.request import Request, urlopen
```

Comment:
If this is a 3rd party library, then we probably also need to add it to requirements.txt, no? If it is not, then these statements should be moved under Standard.

Reply:
nope, built in